templates/solaris-0.1.2/src/Algorithm.py

```diff
@@ -0,0 +1,37 @@
+import Algorithmia
+import solaris as sol
+# API calls will begin at the apply() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+
+def apply(input):
+    return "hello {}".format(str(input))
```

Comment:
I believe the `str` is unnecessary

Reply:
it massages non-string types into something that can work, a copy/paste directly from the anaconda python template. I'm not really sure what a solaris template looks like, but I know this is a customer ask so I have a feeling they know what to do lol